docs/getting-started.md

```diff
@@ -0,0 +1,96 @@
+# Getting Started with Theia
+
+## Table of Contents
+
+<!-- toc -->
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Theia Installation](#theia-installation)
+- [Features](#features)
+  - [Network Flow Visualization and Monitoring](#network-flow-visualization-and-monitoring)
+  - [NetworkPolicy Recommendation](#networkpolicy-recommendation)
+- [Additional Information](#additional-information)
+<!-- /toc -->
+
+## Overview
+
+Theia is a network observability and analytics platform for Kubernetes, bulit
+on top of [Antrea](https://github.com/antrea-io/antrea). Theia consumes [network
+flows exported by Antrea](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to provide fine-grained visibility into the network communications in a
+Kubernetes cluster.
+
+Theia supports network flow visualization and monitoring with Grafana, and can
+recommend appropriate NetworkPolicy configuration to secure Kubernetes network
+and applications. This guide describes how to install and get started with
+Theia.
+
+## Prerequisites
+
+To install Theia, ensure the Flow Exporter feature is enabled in Antrea deployment
+manifest, in the following way:
+
+```yaml
+  antrea-agent.conf: |
+    ...
+    featureGates:
+      ...
+      FlowExporter: true
+```
+
+## Theia Installation
+
+To enable both [Grafana Flow Collector](network-flow-visibility.md) and
+[NetworkPolicy Recommendation](networkpolicy-recommendation.md), please install
+Flow Aggregator and Theia by runnning the following commands:
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
+helm install theia theia/build/charts/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
+```
+
+To enable only Grafana Flow Collector, please install Flow Aggregator and Theia
+by runnning the following commands:
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true -n flow-aggregator --create-namespace
+helm install theia theia/build/charts/theia -n flow-visibility --create-namespace
+```
+
+## Features
+
+### Network Flow Visualization and Monitoring
+
+Theia uses Grafana to visualize network flows in the Kubernetes cluster. After
```

Comment:
Do we need these paragraphs here? We already have the almost the same contents in network-flow-visibility.md L139. 

Shall we make it simple as the NetworkPolicy Recommendation section?

"Theia uses Grafana to visualize network flows in the Kubernetes cluster. Please follow the instructions in the [Network Flow Visibility](network-flow-visibility.md) user guide."

Reply:
The description in network-flow-visibility.md is very detailed. I want list the quick way for users to access Grafana in getting-started.

Reply:
got it. Make sense to me.